### DISCUSSION TITLE
How to implement One To Many Relation?

### DISCUSSION THREAD

**Discussion by godlikeAlex** [filamentphp/filament#8204] [Help] [answered]
hello is it possible to implement One To Many relation in Fillament php?

**Comment by incloudsistemas** [comment]
in many ways 
https://filamentphp.com/docs/3.x/panels/resources/relation-managers

**Comment by godlikeAlex** [comment, marked as answer]
i missed option
in Relation hasMany ->
`protected static ?string $inverseRelationship = 'faq'; `

and for oneToMany used just Select
```
Forms\Components\Select::make("faq_id")->relationship(
  "faq",
  "title"
),
```

**Comment by avinmaster** [comment]
Can you show more of your code?
